src/commands/etherfi.ts

```diff
@@ -239,12 +241,20 @@ async function getClusterSnapshot(
       },
     });
     if (response.status !== 200) throw Error("Request did not return OK");
-    if (!response.data.data.cluster) throw Error("Response is empty");
 
-    let clusterObj = response.data.data.cluster;
-
-    console.debug(`Cluster Snapshot:\n\n${clusterObj.validatorCount}`);
-    clusterSnapshot = clusterObj;
+    if (!response.data.data.cluster) {
```

Comment:
Just noticed on line 211 we already do this assignment. So it's probably better to flip this "if/else" to a simple "if" with reversed logic, and only reassign `clusterSnapshot = response.data.data.cluster` if response.data.data.cluster`.
Basically removes the repetition of the empty object.